docs/designers-developers/developers/slotfills/plugin-block-settings-menu-item.md

```diff
@@ -0,0 +1,26 @@
+# PluginBlockSettingsMenuItem
+
+This slot appears allows for adding a new item in to the More Options area. 
+This will either appear in the controls for each block or at the Top Toolbar depending on the users setting.
+
+
+## Example
+
+```js
+const { registerPlugin } = wp.plugins;
+const { PluginBlockSettingsMenuItem } = wp.editPost;
+
+const PluginBlockSettingsMenuGroupTest = () => (
+	<PluginBlockSettingsMenuItem
+		allowedBlockNames='core/paragraph'
```

Comment:
```suggestion
		allowedBlocks='core/paragraph'
```

This was wrong in the docs and was fixed by https://github.com/WordPress/gutenberg/pull/14741